.github/workflows/pre-release.yml

```diff
@@ -20,22 +20,16 @@ jobs:
         with:
           go-version: '1.18'
 
-      - name: Build
-        uses: goreleaser/goreleaser-action@v4
-        if: ${{ github.event_name == 'pull_request' }}
```

Comment:
We never run this on pull requests, only when we push tags, so this job is useless.